.github/workflows/build_wheels_linux.yml

```
@@ -97,16 +107,34 @@ jobs:
           repository: ${{ inputs.test-infra-repository }}
           ref: ${{ inputs.test-infra-ref }}
           path: test-infra
+      - uses: actions/checkout@v3
+        if: ${{ inputs.architecture == 'ARM64' }}
```

`arm64` usually indicates M1, on Linux it should be `aarch64`
```suggestion
        if: ${{ inputs.architecture == 'aarch64' }}
```